src/BenchmarkDotNet/Environments/HostEnvironmentInfo.cs

```diff
@@ -109,5 +113,22 @@ public override IEnumerable<string> ToFormattedString()
         public bool IsDotNetCliInstalled() => !string.IsNullOrEmpty(DotNetSdkVersion.Value);
 
         private static string GetBenchmarkDotNetVersion() => BenchmarkDotNetInfo.FullVersion;
+
+        [PublicAPI]
+        public static string GetInformation()
+        {
+            var summary = new Summary(
```

Comment:
I don't like an idea to create a fake summary to print information about the environment, it looks like a hack. We have to refactor it somehow.